README.md

```diff
@@ -8,7 +8,34 @@ A Visual Studio Code extension which adds syntax highlighting for Fastly Varnish
 
 ## Features
 
-This uses a JSON [TextMate language grammar](https://macromates.com/manual/en/language_grammars): [syntaxes/vtc.tmLanguage.json](syntaxes/vtc.tmLanguage.json), a structured collection of regular expressions, to tokenize the text into scopes. Visual Studio Code themes map scopes to colours and styles.
+This uses a JSON [TextMate language grammar](https://macromates.com/manual/en/language_grammars): [syntaxes/vtc.tmLanguage.json](syntaxes/vtc.tmLanguage.json), a structured collection of regular expressions, to tokenize the text into scopes such as:
```

Comment:
The link for `syntaxes/vtc.tmLanguage.json` seems broken. Should we replace `vtc` with `vcl`?

Reply:
Otherwise, LGTM.